css-positioning/absolute.css

```diff
@@ -0,0 +1,37 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 235px;
```

Comment:
Such big movements!! You want this in the top right, yes? Moving it all the way from the bottom left seems like a whole lot of work 🤔